aws/rds.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilRdsDeleted(svc *rds.RDS, input *rds.DescribeDBInstancesInput) error {
+	for i := 0; i < 240; i++ {
+		_, err := svc.DescribeDBInstances(input)
+		if err != nil {
+			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "DBInstanceNotFound" {
```

Comment:
NIT: I think AWS has constants in the SDK for the error codes. E.g in [the sdk docs](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/), if you search for this string, you will find a reference to `ErrCodeDBInstanceNotFoundFault`. Not sure where the exact import is though...

Reply:
Looks like the import is just `rds.ErrCodeDBInstanceNotFoundFault`
https://github.com/aws/aws-sdk-go/blob/master/service/rds/examples_test.go#L79 :D